src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -406,10 +368,16 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
         >
           {topStoriesInitialData && (
             <ResponsiveComponentWrapper>
-              <TopStories
-                content={topStoriesInitialData}
-                parentColumns={gridColsSecondary}
-              />
+              {CpsVaried({
+                experimentID: 'test_2',
```

Comment:
I don't think this will work since its on the StoryPage component which won't receive MVT headers.